h2o-core/src/main/java/water/H2ONode.java

```diff
@@ -105,16 +105,24 @@ private UDP_TCP_SendThread startSendThread() {
         _ipLow = ArrayUtils.encodeAsLong(b, 0, 8);
       }
     }
-    public int htm_port() { return getPort()-H2O.ARGS.port_offset; }
-    public int udp_port() { return getPort()  ; }
-    @Override public String toString() { return getAddress()+":"+htm_port(); }
+    /**
+     * Retrieves the public communication port on which the REST API is exposed
+     * @return port number
+     */
+    int api_port() { return getPort()-H2O.ARGS.port_offset; }
```

Comment:
since we are renaming this I would switch it to camelCase as is javaStandard

Reply:
thanks, that makes sense